docs/apm/custom-links.asciidoc

```diff
@@ -0,0 +1,218 @@
+
+[float]
+[[custom-links-url]]
+==== URL
+
+The URL your link points to.
+URLs support dynamic field name variables, encapsulated in double curly brackets: `{{field.name}}`.
+These variables will be replaced with transaction metadata when the link is clicked.
+
+Because everyone's data is different,
+you'll need to examine your own traces to see what metadata is available for use.
+The easiest way to do this is to select a trace in the APM app, and click **Metadata** in the **Trace Sample** table. 
+
+[role="screenshot"]
+image::apm/images/example-metadata.png[Example metadata]
+
+[float]
+[[custom-links-filters]]
+==== Filters
+
+Filter each link to only appear so it only appears for specific services or transactions.
+You can filter on the following fields:
+
+* `service.name`
+* `service.env`
+* `transaction.type`
+* `transaction.name`
+
+Multiple values are allowed when comma separated.
+
+[float]
+[[custom-links-examples]]
+=== Custom link examples
+
+// Relevant documentation links
+:jira-query-params: https://confluence.atlassian.com/jirakb/how-to-create-issues-using-direct-html-links-in-jira-server-159474.html
+:github-query-params: https://help.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters
+
+Not sure where to start with custom links?
+Take a look at the examples below, and customize them to your liking!
+
+[float]
+[[custom-links-examples-email]]
+==== Email
+
+Email the owner of a service.
+
+|====
+|Label |`Email <SERVICE_NAME> engineer`
+|Link |`mailto:<TEAM_OR_ENGINEER>@<COMPANY_NAME>.com`
+|Filters |`service.name:<SERVICE_NAME>`
+|====
+
+**Example**
+
+This link opens an email addressed to the team or owner of `python-backend`.
+It will only appear on services with the name `python-backend`.
+
+|====
+|Label |`Email python-backend engineers`
+|Link |`mailto:python_team@elastic.co`
+|Filters |`service.name:python-backend`
+|====
+
+[float]
+[[custom-links-examples-gh]]
+==== GitHub issue
+
+Open a GitHub issue with pre-populated metadata from the selected trace sample.
+
+|====
+|Label |`Open an issue in <REPO_NAME>`
+|Link |`https://github.com/<ORG>/<REPO>/issues/new?title=<TITLE>&body=<BODY>`
+|Filters |`service.name:client`
+|====
+
+**Example**
+
+This link opens a new GitHub issue in the apm-agent-rum repository.
+It populates the issue body with relevant metadata from the currently active trace.
+Clicking this link results in the following issue being created:
+
+[role="screenshot"]
+image::apm/images/create-github-issue.png[Example github issue]
+
+|====
+|Label |`Open an issue in apm-rum-js`
+|Link |`https://github.com/elastic/apm-agent-rum-js/issues/new?title=Investigate+APM+trace&body=Investigate+the+following+APM+trace%3A%0D%0A%0D%0A---%0D%0A%0D%0Aservice.name%3A+%7B%7Bservice.name%7D%7D%0D%0Atransaction.id%3A+%7B%7Btransaction.id%7D%7D%0D%0Acontainer.id%3A+%7B%7Bcontainer.id%7D%7D%0D%0Aurl.full%3A+%7B%7Burl.full%7D%7D`
```

Comment:
```suggestion
|Link |`https://github.com/elastic/apm-agent-rum-js/issues/new?title=Investigate+APM+trace&body=Investigate+the+following+APM+trace%3A%0D%0A%0D%0A---%0D%0A%0D%0Aservice.name%3A+{{service.name}}%0D%0Atransaction.id%3A+{{transaction.id}}%0D%0Acontainer.id%3A+{{container.id}}%0D%0Aurl.full%3A+{{url.full}}`
```
What I meant was, is it possible to change it to this? That would make the example even more readable with the moustache {}'s in there rather than the unicode characters. AFAIK it still works for Github.

Reply:
Oh snap, this does work, and for Jira too. Sorry about the confusion! I changed the links.